test/TestServerProjectsLowLevel/dpg-customization/Customization/DPGClient.cs

```diff
@@ -101,7 +141,7 @@ public virtual AsyncPageable<Product> GetPagesValuesAsync(string mode, Cancellat
         {
             Argument.AssertNotNull(mode, nameof(mode));
 
-            var context = new RequestContext { CancellationToken = cancellationToken };
+            var context = new RequestContext {CancellationToken = cancellationToken};
```

Comment:
nit: why?

Reply:
Autoformatting

Reply:
Could we put it back the other way?  Pretty sure C# style guidelines require spaces here.

Reply:
Fixed